tests/create-errors/host/host.c

```diff
@@ -36,11 +45,25 @@ static void _test_invalid_param(const char* path, uint32_t flags)
 
     OE_TEST(result == OE_INVALID_PARAMETER);
 
-    /* Content field filled. */
+#ifdef OE_CONTEXT_SWITCHLESS_EXPERIMENTAL_FEATURE
+    /* Invalid configuration with incorrect **config_count** */
+    result = oe_create_create_errors_enclave(
+        path, OE_ENCLAVE_TYPE_SGX, flags, &invalid_config, 0, &enclave);
+
+    OE_TEST(result == OE_INVALID_PARAMETER);
+
+    /* Invalid configuration with correct **config_count** */
+    result = oe_create_create_errors_enclave(
+        path, OE_ENCLAVE_TYPE_SGX, flags, &invalid_config, 1, &enclave);
```

Comment:
invalid_config is not being initialized. Therefore this test is unreliable.

Reply:
Fixed